schemas/documents/shared/credibilityCard.js

```diff
@@ -0,0 +1,36 @@
+export default {
+  title: 'Credibility Card',
+  name: 'credibilityCard',
+  type: 'document',
+  fields: [
+    {
+      title: 'Title',
+      name: 'title',
+      type: 'string',
+      validation: Rule => Rule.required(),
+    },
+    {
+      title: 'Description',
+      name: 'description',
+      type: 'customPortableText',
+      validation: Rule => Rule.required(),
+    },
+    {
+      title: 'Icon',
+      name: 'icon',
+      type: 'image',
+    },
+    {
+      title: 'Claims',
+      name: 'claims',
+      type: 'array',
+      of: [
+        {
+          type: 'reference',
+          to: [{ type: 'claim' }],
+        },
+      ],
+      validation: Rule => Rule.required(),
+    },
```

Comment:
as mentioned in the issue description:

> This should include the on chain project id, the list of credibilityCards and their claims (for this, we'll need to set up a new Sanity object type so we can attach claims to credibility cards at the project level)

I believe it would be preferable to be able to define reusable credibility cards content (without the associated claims) and then map claims to credibility cards at the project level only. 
This way you can reuse the same credibility card content on multiple projects while adding different claims to those projects.

Reply:
I missed that part sorry, this should be better now.
One thing I'm wondering about this structure is how it is displayed in the studio:
![image](https://github.com/regen-network/regen-sanity/assets/530644/382e21f1-012f-4793-8723-4b0606473c99)
Is there a way to make the display preview more helpful?

Reply:
you can use preview option: https://www.sanity.io/docs/previews-list-views
see for instance in `ecologicalImpactRelation`

Reply:
Thanks, I wasn't aware of that option, it looks way better now.
![image](https://github.com/regen-network/regen-sanity/assets/530644/b11b08b3-55e2-47f5-b669-d5ffac5f50ec)